fcp/src/containment/mcp/clients/boundary.py

```
@@ -0,0 +1,16 @@
+"""The boundary of the box."""
+
+from containment.structures import Specification
+from containment.mcp.clients.basic import MCPClient
+from containment.mcp.clients.completion import ImpExpert, ProofExpert
```

Ruff (0.11.9)

`containment.mcp.clients.completion.ImpExpert` imported but unused

Remove unused import

(F401)